src/qibosoq/__main__.py

```diff
@@ -27,9 +27,26 @@ def define_logger():
     return new_logger
 
 
-logger = define_logger()
+def define_program_logger():
+    """Define qick logger format and handler"""
 
-HOST = "192.168.0.72"  # Server address
+    new_logger = logging.getLogger("qick_program")
```

Comment:
Move the name in a top-level constant, or ask it as an input of the function.

Reply:
But I use the name of the logger in another file (`rfsoc_server.py`). 
Should I define all the constants in `__init__.py` and import them around?

Reply:
> Should I define all the constants in `__init__.py` and import them around?

You got the issue. For the time being, just lift to a separate file the constants that you use in multiple places.

I'd suggest avoiding `__init__.py`, because you should have a one-directional flow to avoid circular dependencies.
So, my advice is to `import` only from siblings or descendants (and transitively from descendants of siblings, of course), but not from parents or other files that are not in the same subtree (sometimes is better to give up on this last part, and allow imports from other subtrees, but with a clear hierarchy in mind between them).

So:
```
A/
 b/
  c # can import only from `d`
  d # can import from `c`, if `c` is not importing from it
 e/
  f # can import from `g` and `h/i`
  g # (if really needed, can also import `../b/c`, but then better to
       make sure that no one from inside `b` is importing anything from
       `e`)
  h/
   i # <- can not import from anyone else
```

And about `__init__.py`, you always have to consider it a level above the other modules in the same folder, because it represents the folder itself.

So, I would not put in the global `__init__.py`, in order to keep that just for the global exports.

Reply:
Actually this is a problem also for the main logger, since `__name__` is not a constant between the files... Didn't see it before because of `"__name__"`